docs/getting-started-guides/docker.md

```diff
@@ -0,0 +1,70 @@
+## Running kubernetes locally via Docker
+
+The following instructions show you how to set up a simple, single node kubernetes cluster using Docker.
+
+### Step One: Run etcd
+```sh
+docker run --net=host -d kubernetes/etcd:2.0.5.1 /usr/local/bin/etcd --addr=$(hostname -i):4001 --bind-addr=0.0.0.0:4001 --data-dir=/var/etcd/data
+```
+
+### Step Two: Run the master
+```sh
+docker run --net=host -d -v /var/run/docker.sock:/var/run/docker.sock  gcr.io/google-containers/hyperkube:dev /hyperkube kubelet --api_servers=http://localhost:8080 --v=2 --address=0.0.0.0 --enable_server --hostname_override=127.0.0.1 --config=/etc/kubernetes/manifests
+```
+
+This actually runs the kubelet, which in turn runs a [pod](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/pods.md) that contains the other master components.
+
+### Step Three: Run the service proxy
+*Note, this could be combined with master above, but it requires --privileged for iptables manipulation*
+```sh
+docker run -d --net=host --privileged gcr.io/google_containers/hyperkube:v0.14.1 /hyperkube proxy --master=http://127.0.0.1:8080 --v=2
```

Comment:
In order to do that the kubelet needs to be flag flipped to enable it to run privileged containers.   Not sure if a user expects that (since now anyone with API access can run privileged contaieners)

Will see if we can restrict it only to file manifests.